nimbus-features/onboardingFrameworkFeature.yaml

```diff
@@ -30,7 +30,7 @@ features:
             ALWAYS:                   "true"
             NEVER:                    "false"
           cards:
-            - name: "welcome"
+            wecome:
```

Comment:
Typo? I guess should be welcome
```suggestion
            welcome:
```

Reply:
OOPS! Applying this, but not re-running BR for it.